ClientApp/src/Api/apiClient.ts

```diff
@@ -69,3 +69,12 @@ export async function fetchUnconfirmedSightings(): Promise<null | ListSightings>
   return await response.json();
 }
 
+export async function delSighting(Id:number) {
```

Comment:
can we call this deleteSighting please? It's a big nitpick and not much difference but just to emphasis clear function names :)